### ISSUE TITLE
Gift card purchase unavailable?

### ISSUE THREAD

**Issue by sophie-li** [solidusio/solidus_virtual_gift_card#24]
I installed this extension v1.1.0 with Solidus v1.2, and did not see the ability gift card purchase on the shopping cart. When I installed this extension, the files that are built are only the db migrations and asset files (javascript/spree/backend/all.js and stylesheets/spree/backend/all.css), no html views. However, I able to see the admin functionality to redeem a gift card for a user. 

I even restarted the app, but to no avail.

**Comment by adammathys** [comment]
Purchasing gift cards is no different than purchasing products. (Since gift cards are simply products with a specific column set to true.) Unfortunately, the extension doesn't currently add a way to edit that column in the admin right now.

As a temporary workaround, you can set the column manually in a console:

```ruby
product = Spree::Product.find_by(name: "Adam's Fancy Gift Card")
product.update(gift_card: true)
```

After doing so, everything should work as desired. I'll work on tossing something together to show that field on the admin.

**Comment by adammathys** [comment]
Fixed by #34
